packages/block-library/src/button/block.json

```diff
@@ -89,7 +89,30 @@
 				"radius": true
 			}
 		},
-		"__experimentalSelector": ".wp-block-button__link"
+		"__experimentalSelector": ".wp-block-button__link",
+		"blockStyles": {
+			"border": {
+				"//": "100% causes an oval, but any explicit but really high value retains the pill shape.",
```

Comment:
We should document this approach.

Reply:
This is a "cool" feature that should work across all the JSON files used in WordPress, not only for CSS in JSON.

Reply:
Related to https://github.com/WordPress/gutenberg/issues/35099

Reply:
The feedback in #35099 seems to be that this isn't the best way to do it.